data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/tracing/TracingSpan.java

```diff
@@ -0,0 +1,39 @@
+package dev.knative.eventing.kafka.broker.core.tracing;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.cloudevents.CloudEvent;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.attributes.SemanticAttributes;
+import io.vertx.core.Vertx;
+
+public class TracingSpan {
+
+  static String ACTIVE_CONTEXT = "tracing.context";
+  static String ACTIVE_SPAN = "tracing.span";
+
+  public static final AttributeKey<String> MESSAGING_MESSAGE_SOURCE = stringKey("messaging.message_source");
+  public static final AttributeKey<String> MESSAGING_MESSAGE_TYPE = stringKey("messaging.message_source");
```

Comment:
Should these two be something like `cloudevents.source`?

Reply:
If you see `SemanticAttributes.MESSAGING_MESSAGE_ID` is set to `messaging.message_id`, so I thought it was more clear to keep the same scheme for source and type to avoid any confusion around what they represent.

Reply:
Shouldn't we try to use the attribute keys as the ones used in eventing? https://github.com/knative/eventing/blob/master/pkg/kncloudevents/populate_span_transformer.go

Reply:
Eventing uses messaging.message_id as well: https://github.com/knative/eventing/blob/b3184ba9dcd0d0891a46ec88a7ddfd91f3ae12e9/pkg/tracing/attributes.go#L30

Reply:
:thinking: 
Do we set both `messaging.message_id` and `cloudevents.id` or do we use different keys depending on the component in eventing (which seems a bit inconsistent)?

Reply:
I guess since eventing add both we should set both?

BTW adding both seems a bit nosense, maybe we should file an issue upstream for that?

Reply:
Sure, I'll do some research before opening an issue.

Reply:
https://github.com/knative/eventing/issues/4578